docs/user/faq.rst

```diff
@@ -617,19 +617,17 @@ now simply set attributes on the object:
    # Falcon 2.0
    req.context.cache_backend = MyUltraFastCache.connect()
 
-Since the default context type derives from dict, the existing code will work
-unmodified with Falcon 2.0. Nevertheless, it is recommended to change usage
-as outlined above since the ability to use context as dictionary may be
-removed in a future release.
+The new default context type emulates the dict interface in a way that context
```

Comment:
It's probably OK to talk about the "dict interface" here instead of the "mapping interface", but you may still want to reword it for the sake of consistency. E.g.:

> The new default context type implements a dict-like mapping interface in a way...